index.html

```diff
@@ -57,28 +57,16 @@
     <section id='abstract'>
       <p>
         This specification describes data structures and formats, and a simple
-        processing model, to facilitate card-based payments on the Web. It is
-        used by other specifications to facilitate monetary transactions with a
-        "basic card", such as credit, debit, or prepaid card.
+        processing model, to facilitate payments on the Web with credit, debit,
+        or prepaid cards. It is used by other specifications to facilitate
+        monetary transactions with a "basic card" payment method identifier.
       </p>
     </section>
     <section id='sotd'>
       <p>
-        The working group maintains <a href=
-        "https://github.com/w3c/payment-method-basic-card/issues?utf8=%E2%9C%93&amp;q=is%3Aopen%20is%3Aissue%20-label%3A%22Priority%3A%20Postponed%22%20">
-        a list of all bug reports that the group has not yet addressed</a>.
-        Pull requests with proposed specification text for outstanding issues
-        are strongly encouraged.
+        This is a work in progress and is expected to change as the
+        specification gets implemented and deployed.
```

Comment:
Editorial suggestion:
 "is expected to change with implementation and deployment experience."